library/src/main/java/com/datatorrent/lib/io/fs/FileSplitterInput.java

```diff
@@ -194,7 +194,12 @@ protected boolean processFileInfo(FileInfo fileInfo)
 
   protected void updateReferenceTimes(ScannedFileInfo fileInfo)
   {
-    referenceTimes.put(fileInfo.getFilePath(), fileInfo.modifiedTime);
+    Map<String, Long> referenceTimeMap;
```

Comment:
Nitpicking: please call the local variable ```referenceTimePerInputDir```. Will help to remember why it was changed.